platform/platform-app/src/main/resources/features.xml

```diff
@@ -1217,7 +1217,12 @@
         <feature prerequisite="true">web-container</feature>
         <feature prerequisite="true">platform-dependencies</feature>
         <feature>platform-all</feature>
-        <feature>banana</feature>
+        <!-- banana is currently not working with the existing configuration. It fails to create the
```

Comment:
That was kind of a left over not. I am ok with removing it completely from this file and opening up another ticket to clean up the rest. I cannot leave it here as part of this ticket since it now fails all the time anyway.
⭕️

Reply:
I am leaving it as is for this ticket and opened a new ticket to track the banana removal (https://codice.atlassian.net/browse/DDF-3780).